src/stylesheets/_variables.scss

```diff
@@ -0,0 +1,13 @@
+// Colors
+$white: #fff !default;
```

Comment:
@rafayet-monon Later on, it will have a lot of colors.
I recommend you to turn these to the band of color.
```scss
$color-gray: (
  primary: hsl(204, 15%, 87%),
  light: hsl(206, 28%, 95%),
  lighter: hsl(204, 45%, 98%),
  lightest: hsl(210, 50%, 98%),
  dark: hsl(258, 100%, 10%)
);
```

You can change it later. 🤣